From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthew Jeffryes <mjeffryes@pulumi.com>
Date: Mon, 6 May 2024 16:50:17 -0700
Subject: [PATCH] Add shim


diff --git a/shim/shim.go b/shim/shim.go
new file mode 100644
index 00000000..3115399c
--- /dev/null
+++ b/shim/shim.go
@@ -0,0 +1,12 @@
+package shim
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	mist "github.com/Juniper/terraform-provider-mist/internal/provider"
+)
+
+const version = "dev"
+
+func Provider() provider.Provider {
+	return mist.New()()
+}
